app/controllers/idv/image_uploads_controller.rb

```diff
@@ -0,0 +1,52 @@
+module Idv
+  class ImageUploadsController < ApplicationController
+    include ApplicationHelper # for liveness_checking_enabled?
+
+    respond_to :json
+
+    def create
+      image_form = Idv::ApiImageUploadForm.new(
+        params,
+        liveness_checking_enabled: liveness_checking_enabled?,
+      )
+
+      form_response = image_form.submit
+
+      if form_response.success?
+        doc_response = doc_auth_client.post_images(
+          front_image: image_form.front.read,
+          back_image: image_form.back.read,
+          selfie_image: image_form.back&.read,
```

Comment:
should this be `image_form.selfie`?

Reply:
Yes, great catch, thanks 🤦

Reply:
Fixed in fd4d72ac842e718b5e50d69e76e1f182a326c36c